pages/Useful Utilities/Status-Bars.md

```diff
@@ -18,7 +18,7 @@ sudo ninja -C build install
 ```
 
 If you want to use the workspaces module, first, copy the configuration files from
-`/etc/xdg/waybar/` into `~/.config/waybar/`. Then, in `~/.config/waybar/conf/` replace
+`/etc/xdg/waybar/` into `~/.config/waybar/`. Then, in `~/.config/waybar/waybar/config` replace
```

Comment:
not `~/.config/waybar/config`??

Reply:
thank you for pointing that out @vaxerski , just checked my shell histfile.
i had made the `~/.config/waybar` directory and then copied the `/etc/xdg/waybar` directory inside it, hence the confusion.

either way, let me make another commit fixing the path.